.rails-lens.yml

```diff
@@ -0,0 +1,8 @@
+# .rails-lens.yml (optional)
+adapter: auto  # auto-detects PostgreSQL/MySQL/SQLite3
+marker_format: rails-lens:schema  # LLM-friendly markers
```

Comment:
the marker_format will be removed in the next patch version, it better to have consistent across the ecosystem, than having every project creating their own confusing marker.


If you have a better naming, i'm all hears.